etc/fibo-vue/src/views/Courses.vue

```diff
@@ -1,313 +1,96 @@
-
-<h5><span>The course is geared toward</span></h5>
-<ul>
-<li>Data architects</li>
-<li>Data application programmers</li>
-</ul>
-
-<h5><span>Prerequisites</span></h5>
-<ul>
-<li><a href="#KG-M2">Module 2: Knowledge Graph Enterprise Framework</a>, or</li>
-<li>Working knowledge of RDF, SPARQL and OWL</li>
-</ul>
-
-<h5><span>Course outline</span></h5>
-<ol>
-<li>Converting data to Graphs</li>
-<ol>
-<li>Tables (Spreadsheets)</li>
-<li>Tables (Relational Databases)</li>
-<li>XML</li>
-<li>JSON</li>
-</ol>
-<li>Mapping data to ontologies</li>
-<ol>
-<li>Reification of data</li>
-<li>Blank nodes and reified data</li>
-<li>Roles and Individuals, values and data</li>
-</ol>
-<li>Mapping examples:</li>
-<ol>
-<li>Interest rate swaps</li>
-<li>Legal Entity Identifiers</li>
-</ol>
-<li>Exercises: Queries over graph data</li>
-<ol>
-<li>Introduction to query platform</li>
-<li>Queries: Exploring the data</li>
-<li>Queries; Simplifying ownership data</li>
-<li>CONSTRUCT queries: Creating new graph data</li>
-<li>Queries: Who trades with Whom</li>
-<li>Queries: Ultimate parents</li>
-<li>Queries: Counterparty exposure</li>
-<li>&ldquo;Crosswalk&rdquo; data</li>
-</ol>
-</ol>
-
-      </article>
-    </main>
-  </div>
-</template>
-
-<script>
-import helpers from '../store/helpers.js';
-import { outboundClick, outboundLinkClick } from '../helpers/ga';
-
-export default {
-  extends: helpers,
-  name: 'courses',
-  components: {},
-  methods: {
-    outboundClick,
-    outboundLinkClick,
-  },
-};
-</script>
-
-<style lang="scss" scoped>
-</style>
+<template>
+  <div class="container">
+    <main>
+      <article>
+<div>
+<h1>
+<span>Invitation</span>
+</h1>
+<div style="text-align: justify;">
+<p>
+Businesses today need to have comprehensive control and understanding of the data both within the enterprise and from outside. Regulatory pressure in many industries requires that an enterprise be able to find all relevant data about customers, products, business relations and in fact any aspect of the business quickly and accurately. Advances in data analytics have shown that the more meaningful data an enterprise can bring to bear, the more accurately business drivers can be predicted. For this reason, many businesses are migrating from viewing their enterprise information as a collection of data to viewing it as an ongoing graph of meaningful knowledge.
+</p>
+</div>
+<h1>
+<span>Offer</span>
+</h1>
+<div style="text-align: justify;">
+<p>EDMC offers courses in Knowledge Graph, available in multiple forms:</p>
+<ol>
+<li>Instructor-led remote instruction by one of the EDM Council&rsquo;s certified trainers. These courses are open enrollment, registration at the EDM Council web site,</li>
+<li>Enterprise focused instruction. The same course material and instructor as our open enrollment course, but with the audience limited to knowledge workers from your organization. This allows more specific focus on use cases and industries of interest to your needs.</li>

```

Comment:
should make it clearer what prerequisite knowledge or background is expected, especially given it involves exercises. Even saying it's targeted at "knowledge workers" does not help much. Speaking of exercises, should say what software/hardware requirements are.

Reply:
In our wiki, for this course, we say "This is for data professionals.  This includes DBA's, data scientists, analysts, and even the more technical business architects.  Someone who knows a good data model when they see one, and knows why it is important to have a good data model. "  That is pretty indicative, but not good copy for a web site. 

How about this (paraphrased from the wiki to be more in line with web page copy standards): 

This course is targeted to data professionals.  This includes Database Administrators, data scientists, analysts, as well as business architects and product managers for data-driven applications.  In general, this course is for anyone who knows a good data model when they see it, and who knows why that is important. 



I think that should go right before the bullets that break out the content.

Reply:
The exercises are on data.world, so there are only negative requirements; that is, that there is NOT a firewall that blocks access to data.world.